pkg/v1/tkg/test/tkgctl/aws_cc/aws_cc_ccb_test.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2021 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// nolint:typecheck,nolintlint
+package aws_cc
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo"
+
+	. "github.com/vmware-tanzu/tanzu-framework/pkg/v1/tkg/test/tkgctl/shared"
+)
+
+var _ = Describe("Functional tests to create AWS cluster with custom ClusterBootstrap", func() {
+	E2ECommonSpec(context.TODO(), func() E2ECommonSpecInput {
+		return E2ECommonSpecInput{
+			E2EConfig:       e2eConfig,
+			ArtifactsFolder: artifactsFolder,
+			Cni:             "antrea",
+			Plan:            "dev",
+			Namespace:       "tkg-system",
+			OtherConfigs:    map[string]string{"clusterclass": "true"},
```

Comment:
You need to pass in something to say to execute custom clusterbootstrap. May be a boolean flag or pass in the customboostrap cc file

Reply:
added a boolean flag to control